calico_kubernetes/calico_kubernetes.py

```diff
@@ -708,7 +590,7 @@ def _label_to_tag(self, label_key, label_value):
         logger.info("Using CALICOCTL_PATH=%s", CALICOCTL_PATH)
         logger.info("Using KUBE_API_ROOT=%s", KUBE_API_ROOT)
         logger.info("Using CALICO_IPAM=%s", CALICO_IPAM)
-        logger.info("Using DEFAULT_POLICY=%s", DEFAULT_POLICY)
+        logger.info("Using CALICO_POLICY=%s", CALICO_POLICY)
```

Comment:
We should change this to match the `POLICY_ONLY_CALICO` variable used by calico/node, as seen here: https://github.com/projectcalico/calico-docker/blob/master/node_filesystem/etc/rc.local#L5